connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheConnector.java

```diff
@@ -249,6 +251,34 @@
             }
         }
 
+        Object redirectStrategy = config.getProperties().get(ApacheClientProperties.REDIRECT_STRATEGY);
+        if (redirectStrategy != null) {
+            if (!(redirectStrategy instanceof RedirectStrategy)) {
```

Comment:
instanceof is null safe operation, so I would advise to eliminate NULL check here (even though in original connector there are such a redundant checks). 

same is valid for ``defaultAuthSchemeRegistry`` NULL check